.github/workflows/cicd.yml

```diff
@@ -10,7 +10,7 @@ on:
 
 jobs:
   build_and_test:
-    name: CICD tests
```

Comment:
Did changing this name mess up anyones work flow?

Reply:
Good question. Afaik this just changes what is displayed in the githib UI. @raghu-sarangapani do you know if this is ok to change?

Reply:
It should be fine, we'll find out :)